tests/fixtures/trustyai.py

```diff
@@ -51,3 +59,84 @@ def patched_dsc_lmeval_allow_all(
         trustyai_operator_deployment.scale_replicas(replica_count=num_replicas)
         trustyai_operator_deployment.wait_for_replicas()
         yield dsc
+
+
+@pytest.fixture(scope="class")
+def dataset_pvc(admin_client, model_namespace) -> Generator[PersistentVolumeClaim, None, None]:
+    """
+    Creates a PVC for holding dataset files.
+    """
+    pvc_kwargs = {
+        "name": "my-pvc",
+        "namespace": model_namespace.name,
+        "client": admin_client,
+        "size": "1Gi",
+        "accessmodes": "ReadWriteOnce",
+        "label": {"app.kubernetes.io/name": "dataset-storage"},
+    }
+
+    with PersistentVolumeClaim(**pvc_kwargs) as pvc:
+        yield pvc
```

Comment:
No need to define `pvc_kwargs`, you can just pass the arguments to `PersistentVolumeClaim`. Creating a variable for kwargs makes sense if the kwargs can change, but in this case they're fixed:
```suggestion
    pvc_kwargs = {
        "name": "my-pvc",
        "namespace": model_namespace.name,
        "client": admin_client,
        "size": "1Gi",
        "accessmodes": "ReadWriteOnce",
        "label": {"app.kubernetes.io/name": "dataset-storage"},
    }

    with PersistentVolumeClaim(name="my-pvc", namespace=model_namespace.name,...) as pvc:
        yield pvc
```